packages/manager/modules/pci/src/projects/project/workflow/add/schedule/schedule.controller.js

```diff
@@ -0,0 +1,81 @@
+import random from 'lodash/random';
+import {
+  DEFAULT_CRON_PATTERN_DOM,
+  DEFAULT_CRON_PATTERN_MONTH,
+  DEFAULT_CRON_PATTERN_DOW,
+  SCHEDULE_ENUM,
+} from './schedule.constants';
+import CronValidator from './cron-validator';
+
+export default class {
+  $onInit() {
+    this.minutesInvalid = false;
+    this.hoursInvalid = false;
+    this.domInvalid = false;
+    this.monthInvalid = false;
+    this.dowInvalid = false;
+    this.cronValidator = new CronValidator();
+    this.WORKFLOW_SCHEDULES = [{
```

Comment:
This should be in a `constant` file maybe ?

Reply:
this is an object, most of properties values are coming from constants.